libfuzzer-dotnet-windows.cc

```diff
@@ -134,7 +137,11 @@ FUZZ_EXPORT int __cdecl LLVMFuzzerInitialize(int *argc, char ***argv)
         die_sys("CreatePipe() failed");
     }
 
-    TCHAR sharedMemName[] = TEXT("LIBFUZZER_DOTNET_SHMEM");
+    UUID uuid;
+    UuidCreate(&uuid);
+    TCHAR *sharedMemName;
+    UuidToString(&uuid, (unsigned char **) &sharedMemName);
```

Comment:
Hm, I'm getting an error building this in my Visual Studio. Maybe it would be best to explicitly use the Unicode version and do something like this?

```cpp
UuidToStringW(&uuid, (RPC_WSTR)*&sharedMemName);
```